File: unit8/java/store/Simulation.java
import java.util.Scanner;

public class Simulation {

    public static void main(String[] args) {
        // TODO: Create stores for both order plans

        // Read in simulated purchase data
        Scanner scan = new Scanner(System.in);
        while (scan.hasNextLine()) {
            String purchase = scan.nextLine();
            // TODO: Replace with your store objects
            // boolean wasSold = simulateSale(store, purchase);
        }

        // TODO: summarize results
    }

    public static boolean simulateSale(Store store, String purchase) {
        String[] data = purchase.split(",");
        String id = data[0];
        int amount = Integer.parseInt(data[1]);
        double payment = Double.parseDouble(data[2]);
        return store.sell(id, amount, payment);
    }

}

File: unit8/python/store/simulation.py
def simulate_sale(store: Store, purchase: str) -> bool:
    data = purchase.split(',')
    id = data[0]
    amount = int(data[1])
    payment = float(data[2])
    return store.sell(id, amount, payment)


def main() -> None:
    # TODO: Create stores for both order plans

    # Read in simulated purchase data
    while True:
        try:
            purchase = input()
            # TODO: Replace with your store objects
            # was_sold = simulate_sale(store, purchase)
        except EOFError:
            break

    # TODO: summarize results


if __name__ == '__main__':
    main()
